pom.xml

```diff
@@ -28,6 +28,12 @@
 
   <properties>
     <findbugs.failOnViolation>true</findbugs.failOnViolation>
+    <jacoco.agent.line/>
+    <surefire.argLine>
+      -Dfile.encoding=${project.build.sourceEncoding}
+      ${jacoco.agent.line}
+    </surefire.argLine>
+    <version.jacoco.plugin>0.8.2</version.jacoco.plugin>
```

Comment:
Can we only overrides these properties in the jacoco profile?

Reply:
+1, even as just a temporary solution - it will end up in kie-parent